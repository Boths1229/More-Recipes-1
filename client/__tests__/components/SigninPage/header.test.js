import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../../components/SigninPage/Header';

describe('Header Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.find('nav').length).toBe(1);
    expect(wrapper.find('button').length).toBe(1);
    expect(wrapper.find('header').length).toBe(1);
    expect(wrapper).toMatchSnapshot();
  });
});
