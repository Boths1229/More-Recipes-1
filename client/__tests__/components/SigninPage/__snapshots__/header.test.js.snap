// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render correctly 1`] = `
<header>
  <nav
    className="navbar navbar-expand-lg navbar-dark"
    style={
      Object {
        "backgroundColor": "#2b3034",
      }
    }
  >
    <a
      className="navbar-brand"
      href="/"
    >
      More-Recipes
    </a>
    <button
      aria-controls="navbarSupportedContent"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarSupportedContent"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarSupportedContent"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <li
          className="nav-item"
        >
          <Link
            className="nav-link"
            onlyActiveOnIndex={false}
            style={Object {}}
            to="/signup"
          >
            Signup
          </Link>
        </li>
      </ul>
    </div>
  </nav>
</header>
`;
